etc/config/fortran.amazon.properties

```diff
@@ -0,0 +1,114 @@
+compilers=&gfortran_86:&gfortran_arm:&gfortran_ppc
+defaultCompiler=gfortran81
+demangler=/opt/compiler-explorer/gcc-8.1.0/bin/c++filt
+objdumper=/opt/compiler-explorer/gcc-8.1.0/bin/objdump
+
+###############################
+# GCC for x86
+group.gfortran_86.compilers=gfortran412:gfortran447:gfortran453:gfortran464:gfortran471:gfortran472:gfortran473:gfortran474:gfortran481:gfortran482:gfortran483:gfortran484:gfortran485:gfortran490:gfortran491:gfortran492:gfortran493:gfortran494:gfortran510:gfortran520:gfortran530:gfortran540:gfortran550:gfortran6:gfortran62:gfortran63:gfortran71:gfortran72:gfortran73:gfortran81:gfortransnapshot
+group.gfortran_86.groupName=GCC x86-64
+compiler.gfortran412.exe=/opt/compiler-explorer/gcc-4.1.2/bin/gfortran
+compiler.gfortran412.name=x86-64 gfortran 4.1.2
+compiler.gfortran447.exe=/opt/compiler-explorer/gcc-4.4.7/bin/gfortran
+compiler.gfortran447.alias=/usr/bin/gfortran-4.4
```

Comment:
Please remove the `alias` settings here: these were only introduced to work around some very old URLs that had the full path to old C++ compilers in them. We no longer make those URLs, and I don't want to perpetuate the aliasing stuff. Sorry for the confusion: I suspect you sensibly copied them from the C++ properties!

Reply:
I've deleted the alias in the files. Wasn't quite sure how they're supposed to work anyway.